config/environment.default.js

```diff
@@ -8,13 +8,13 @@ module.exports = {
     nameSpace: '/'
   },
   // The REST API server settings.
-  rest: {
+    rest: {
```

Comment:
This indentation is slightly odd. Looks to be accidental?  (There's another such indentation in this same config file)